runtime/src/runtime/fvm.rs

```diff
@@ -231,14 +232,15 @@ where
         rand_epoch: ChainEpoch,
         entropy: &[u8],
     ) -> Result<[u8; RANDOMNESS_LENGTH], ActorError> {
-        fvm::rand::get_chain_randomness(personalization as i64, rand_epoch, entropy).map_err(|e| {
+        let digest = fvm::rand::get_chain_randomness(personalization as i64, rand_epoch, entropy).map_err(|e| {
             match e {
                 ErrorNumber::LimitExceeded => {
                     actor_error!(illegal_argument; "randomness lookback exceeded: {}", e)
                 }
                 e => actor_error!(assertion_failed; "get chain randomness failed with an unexpected error: {}", e),
             }
-        })
+        })?;
+        Ok(draw_randomness(&digest, personalization, rand_epoch, entropy))
```

Comment:
I was initially intending to do this in the SDK but... yeah, it actually makes sense to do it here given that the FVM doesn't really care about _how_ the user draws randomness anyways.

So LGTM?

Reply:
Yeah, this seems like the better thing to do? It does mean that all contracts need to implement their own "drawing" logic, but that's a good thing, arguably.